### DISCUSSION TITLE
スマホでQiitaのホーム画面の「もっと読む」を押した際の挙動について

### DISCUSSION THREAD

**Discussion by yoshi389111** [increments/qiita-discussions#400] [要望・アイデア・不具合]
PC版でQiitaのホーム画面の下部にある「もっと読む」ボタンを押した場合には、続きが下に追加されると思います（「もっと読む」ボタンは画面表示範囲よりも下の方に移動する）。

スマホ版（あるいはPCでブラウザを狭くして表示した状態）で同様に「もっと読む」ボタンを押した場合には、「もっと読む」ボタンが表示されたままで、その上に記事が挿入されるようになっていると思います。

スマホ版でも通常のPC版（画面を広く表示した場合）と同様の挙動にはできないでしょうか。

【PC版で「もっと読む」ボタンを押す前】

![pc_before](https://user-images.githubusercontent.com/15214393/209656874-06975435-d099-465f-afb2-bf326e4c558f.png)

【PC版で「もっと読む」ボタンを押した後】

![pc_after](https://user-images.githubusercontent.com/15214393/209656938-92adfc9c-e12b-499e-a32b-36b2515f3b13.png)

【スマホ版で「もっと読む」ボタンを押す前】

![sp_before](https://user-images.githubusercontent.com/15214393/209657028-e58149af-fd1b-47fd-bf5d-34e9028328da.png)

【スマホ版で「もっと読む」ボタンを押した後】

![sp_after](https://user-images.githubusercontent.com/15214393/209657036-db672c0c-d4c0-487d-bebf-7f6e6568273d.png)

**Comment by tanaka-asami415** [comment]
@yoshi389111 
ご報告ありがとうございます！
弊社で確認したところ、再現する環境と再現しない環境があるようです。
もしよろしければ、ご使用の環境（デバイス、OSのバージョン、ブラウザの種類とバージョン）
を教えていただけないでしょうか🙇

**Reply by yoshi389111** [reply]
発生している環境は以下の通りです。

* Android12(Pixel3a) Chrome 108.0.5359.128
* Windows11 Chrome 108.0.5359.125

ちなみに、Windowsの場合、ウィンドウの幅でプログレッシブなデザインになっていると思いますが、以下のようになっています。

* 1200px以上：下に追加（本事象は発生せず）
* 992px以上：下に追加（本事象は発生せず）
* 991px以下：上に挿入（本事象が発生）

**Reply by kskwtnk** [reply]
@yoshi389111 
ありがとうございます！
いただいた内容も踏まえ、調査してみます 🙏

**Reply by yoshi389111** [reply]
本件、しばらく前から再現しなくなっている（解決している）ようです。
（スマホも、PCで横幅を狭くした状態でも）
何か別件での対応で解決したんですかね。

特に問題が無いようであれば、クローズしてもらっても大丈夫だと思います。

ありがとうございました。

**Comment by kimkim0814** [comment]
@yoshi389111
ご報告を、失念しておりました！
先日、スマホでQiitaのホーム画面の「もっと読む」を押した際の挙動について修正いたしました:blush:
ご不便おかけいたしました。
